tools/run_tests/xds_k8s_test_driver/framework/xds_flags.py

```diff
@@ -173,6 +173,12 @@
     ),
 )
 
+ENABLE_GFE_DEBUG_HEADER = flags.DEFINE_bool(
```

Comment:
Let's make it a `DEFINE_enum` with possible options `"gfe_response_only", "all_response", "request_and_response"`

Reply:
I think we'll need another value to disable adding debug header.